c2cgeoportal/scripts/c2cupgrade.py

```diff
@@ -382,8 +375,12 @@ def files_to_remove(self, summary=False):
                         error = True
                         managed = True
                 if not managed:
-                    print(colorize("The file '{}' is removed.".format(file_), GREEN))
-                    if not summary:
+                    if not pre:
```

Comment:
this "if" can be merged with the previous one no ? (not important)

Reply:
Thanks, fixed (all) :-)